src/composed/key/builder.rs

```diff
@@ -1756,7 +1727,7 @@ mod tests {
         }
     }
     #[cfg(feature = "draft-pqc")]
-    fn gen_ed448_ml_kem_x448<R: Rng + CryptoRng>(mut rng: R, version: KeyVersion) {
+    fn gen_ed448_ml_kem_x448<R: RngCore + CryptoRng + ?Sized>(rng: &mut R, version: KeyVersion) {
```

Comment:
do we have to switch to `&mut` ? I am really not a big fan of forcing a reference